serverless.yml

```diff
@@ -0,0 +1,42 @@
+service: sqstodynamo
+
+provider:
+    name: aws
+    region: us-east-1
+    runtime: provided
+    iamRoleStatements:
+        - Effect: Allow
+          Action:
+              - sqs:SendMessage
+              - sqs:ReceiveMessage
+              - sqs:DeleteMessage
+              - sqs:GetQueueAttributes
+              - logs:CreateLogGroup
+              - logs:CreateLogStream
+              - logs:PutLogEvents
+          Resource: '<your-sqs-arn>'
+        - Effect: Allow
+          Action:
+              - dynamodb:BatchGetItem
+              - dynamodb:GetItem
+              - dynamodb:Query
+              - dynamodb:Scan
+              - dynamodb:BatchWriteItem
+              - dynamodb:PutItem
+              - dynamodb:UpdateItem
+          Resource: '<your-dynamodb-table-arn>'
+
+plugins:
+    - ./vendor/bref/bref
+
+functions:
+    hello:
```

Comment:
This name could be more clear on what the function does. For a "hello world" app it kind of makes sense, but this is shuffling data around.

Reply:
Agreed. Will refactor it.